tests/aws/services/apigateway/test_apigateway_basic.py

```diff
@@ -2012,108 +1965,6 @@ def test_api_gateway_http_integrations(
         assert expected == content["data"]
         assert ctype == headers["content-type"]
 
-    @markers.aws.unknown
-    def test_api_gateway_kinesis_integration(
```

Comment:
this was already tested in `test_apigateway_kinesis.py`, right?

Reply:
Yes there are 5 tests that were already validated, I captured it with the commit message as I commited 1 test per commit.
- removed `test_malformed_response_apigw_invocation` tested by `test_apigateway.test_apigateway_lambda.py::test_lambda_aws_proxy_response_format`
- `test_api_gateway_update_resource_path_part` removed as covered in `test_apigateway.TestApiGatewayApiResource.test_resource_lifecycle`
- `test_api_gateway_kinesis_integration` removed already validated with `test_apigateway_kinesis.py::test_apigateway_to_kinesis`
- `test_api_gateway_sqs_integration_with_event_source` removed as validated by `test_apigateway_sqs.py`
- `test_http_integration` removed as testing same functionality as `test_apigateway_http.test_http_integration_with_lambda`

Reply:
Ah right, sorry, haven't really read the commit messages 😅